pkg/apis/options/cookie.go

```diff
@@ -0,0 +1,15 @@
+package options
+
+import "time"
+
+// CookieOptions contains configuration options relating to Cookie configuration
+type CookieOptions struct {
```

Comment:
Does each field need to begin with `Cookie`? If these fields are explicitly related to a Cookie then it's a bit of unneeded duplication imo.

Reply:
It doesn't necessarily no, I had done this so that I didn't have to find and replace all of the occurrences of each field to minimize diffs in this PR. Also, at present this `CookieOptions` struct is embedded in the `Options` struct, if we rename these fields we probably want to get rid of this embedding, that would likely result in a change to the configuration file layout, so perhaps leave that for a later configuration oriented refactor? I have plans to replace the existing config management with viper so that we can create groups of flags and config options and compose them together for the full configuration struct

Reply:
Fine with me.